test/lit/test/basic.test.ts

```diff
@@ -9,6 +9,19 @@ declare global {
 
 describe('Button with increment', async() => {
   beforeEach(async() => {
+    Object.defineProperty(Element.prototype, 'localName', {
```

Comment:
Can you add a TODO comment linking to the PR you create and we can remove this after it's been merged?

Reply:
uhmm, do you mean the PR to happy-dom? wait I need to make the PR first ubt @patak-dev want to wait...

Reply:
@antfu wants to merge it 👍🏼